oak_containers_stage1/src/main.rs

```diff
@@ -49,22 +59,15 @@ async fn main() -> Result<(), Box<dyn Error>> {
         MsFlags::empty(),
         None::<&str>,
     )
-    .context("overmounting root")?;
-
-    // musl expects /proc to be mounted, otherwise libc calls such as realpath(2) will fail.
-    create_dir("/proc")?;
-    mount(
-        None::<&str>,
-        "/proc",
-        Some("proc"),
-        MsFlags::MS_NOEXEC | MsFlags::MS_NOSUID | MsFlags::MS_NODEV,
-        None::<&str>,
-    )?;
+    .context("Error overmounting a ramfs on root")?;
```

Comment:
nit: our convention is to start anyhow context strings with lowercase letters, also below